lib/common.sh

```diff
@@ -41,6 +41,8 @@ CONTAINER_RUNTIME=${CONTAINER_RUNTIME:-"podman"}
 # Hostname format
 NODE_HOSTNAME_FORMAT=${NODE_HOSTNAME_FORMAT:-"node-%d"}
 
+CAPM3_RELEASE=${CAPM3_RELEASE:-"v0.3.1"}
```

Comment:
is there a way to retrieve this automatically from Github ? otherwise we will need to update it everytime we release a version

Reply:
Done